spec/requests/api/requests_spec.rb

```diff
@@ -228,6 +228,22 @@
       expect(response.parsed_body).to include(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes workflow in the request resources" do
+      ems = FactoryGirl.create(:ems_vmware)
+      vm_template = FactoryGirl.create(:template_vmware, :name => "template1", :ext_management_system => ems)
+      request = FactoryGirl.create(:miq_provision_request, :requester => @user, :src_vm_id => vm_template.id, :options => {:owner_email => 'tester@miq.com'})
+      workflow_class = request.workflow_class
+      allow_any_instance_of(workflow_class).to receive(:get_dialogs).and_return(:dialogs => {})
```

Comment:
This `any_instance_of` is unfortunate - is there any way it can be  avoided? What happens when it's not stubbed?